### ISSUE TITLE
how to refined the super parameters

### ISSUE THREAD

**Issue by zfs1993** [sirius-ai/MobileFaceNet_TF#8]
i try the work you have done,it works,but i did not get the precision you showed,can you give me some advices. thank you very much

**Comment by sirius-ai** [comment]
what dataset are you use？And what hyparameters config in your project?

**Comment by zfs1993** [comment]
the data you provide,default parameters in project, i see the result in tensorboard and find that inference loss is shaking from epoch 3 ,getting bigger and bigger. i thought it may caused by the learning rate doesn't change to the smaller one in time.

**Comment by sirius-ai** [comment]
In my test, reg_loss raise in begin but as the trainning go on it will be decrease，inference-loss aways trend to minish.

**Comment by sirius-ai** [comment]
weight decay is use for Lp regularization， and reg_loss is the sum of regularization loss which collect in tf.GraphKeys.REGULARIZATION_LOSSES, total loss is the sum of normal loss and reg loss.

**Comment by chenkingwen** [comment]
it seemed that the purpose of weight decay and reg_loss are same? because the reg_loss gonna decrease the weights value,  is that right? @sirius-ai

**Comment by sirius-ai** [comment]
In my opinion，not only weight decay but also regularizer can control the reg loss. On the whole, all of them effect the training performance.

**Comment by sirius-ai** [comment]
close. when it necessary will reopen.
